src/index.html

```diff
@@ -8,13 +8,17 @@
     <h1>Zigbee and Thread USB stick by the creators of Home Assistant</h1>
     <p>
       Home Assistant SkyConnect is the easiest way to add Zigbee support to your
-      Home Assistant instance and make it Matter-ready. It features a transmit
-      power of up to 20 dBm and over-the-air updates. A future firmware update
-      will bring Thread support; allowing SkyConnect to power your Matter and
-      Zigbee networks at the same time.
+      Home Assistant instance and make it Matter-ready. Home Assistant SkyConnect features a transmit
+      power of up to 20 dBm and over-the-air updates.
     </p>
+    <h3>Thread and Matter</h3>
+    <p> A future firmware update
+      will bring Thread support; allowing SkyConnect to power your Matter and
+      Zigbee networks at the same time.</p>
+    <h3>Zigbee Home Automation (ZHA)</h3>
```

Comment:
I'd just use Zigbee here, so

```suggestion
    <h3>Zigbee</h3>
```

Also, can you move thiss before Thread and Matter? (along with the Choice below, effectively move the Thread Matter to the bottom of the page).